content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: OTel Operator Q&A
+linkTitle: OTel Operator Q&A
+date: 2024-04-24
+author: >-
+  [Adriana Villela](https://github.com/avillela) (ServiceNow),
+
+canonical_url: https://adri-v.medium.com/81d63addbf92?
+cSpell:ignore: automagically mycollector
+---
+
+![Seattle's Mount Rainier rising about the clouds, as seen from an airplane. Photo by Adriana Villela](mount-rainier.jpg)
+
+The
+[OpenTelemetry (OTel) Operator](https://github.com/open-telemetry/opentelemetry-operator)
+is a
+[Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+that manages OTel things for you in your Kubernetes cluster to make life a
```

Comment:
```suggestion
that manages OTel for you in your Kubernetes cluster to make your life a
```

Unless we can specify what "things" are.

Reply:
I've added that to the post. BTW, I'm a maintainer on the End User SIG, so if you'd like us to work with you to get community feedback on the Operator, let me know! We can run surveys and end user panels.